packages/bootstrap-4/test/Array.test.tsx

```diff
@@ -47,4 +68,32 @@ describe("array fields", () => {
       .toJSON();
     expect(tree).toMatchSnapshot();
   });
+  test("customization", () => {
```

Comment:
make this test name more descriptive, so it's clear it's about description field customization with a fixed array

Reply:
done!